sortinghat/core/management/commands/create_user.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2014-2022 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     Jose Javier Merchante <jjmerchante@bitergia.com>
+#
+
+import logging
+
+import django.db
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand
+
+User = get_user_model()
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Create new a new account in SortingHat"
+
+    def add_arguments(self, parser):
+        parser.add_argument("username", help="Account username")
+        parser.add_argument("password", help="Account password")
```

Comment:
I wonder if we can use the same parser that django-admin uses when a new superuser is created.